src/api/rails.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const rails = axios.create({
+  baseURL: 'http://localhost:5000/api',
```

Comment:
I would have used an environment file to store URL's. URL's may differ from environment to environment.

Reply:
I agree. Thanks for the tip